packages/boxel-ui/addon/src/components.ts

```diff
@@ -0,0 +1,58 @@
+import Accordion from './components/accordion/index.gts';
+import AddButton from './components/add-button/index.gts';
+import Button from './components/button/index.gts';
+import BoxelButton from './components/button/index.gts';
+import CardContainer from './components/card-container/index.gts';
+import BoxelDropdown from './components/dropdown/index.gts';
+import DropdownButton from './components/dropdown-button/index.gts';
+import FieldContainer from './components/field-container/index.gts';
+import GridContainer from './components/grid-container/index.gts';
+import BoxelHeader from './components/header/index.gts';
+import Header from './components/header/index.gts';
+import IconButton from './components/icon-button/index.gts';
+import BoxelInput from './components/input/index.gts';
```

Comment:
so for this new style, does VS code still understand these imports, and is glint still giving us types within vs code for these imports?

Reply:
Yep!

Reply:
> Also, if we fail to import with the file extension, will linting warn us we are doing it wrong? (and/or are build errors are clear enough to guide us to the correct action)

There is definitely room for improvement here.